relex.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import asyncio
+import random
+import re
+from discord.ext import commands
+from dotenv import load_dotenv
+from googlesearch import search
+
+from newest_xkcd import latest_comic_num
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+
+URL = 'https://xkcd.com/'
+
+
+bot = commands.Bot(command_prefix='!')
+
+@bot.command(name='number', pass_context=True)
+async def number(ctx, number):
+    await ctx.send(f'{URL}{number}/')
+
+@bot.command(name='random_comic', pass_context=True)
+async def random_comic(ctx):
+    newest_comic = latest_comic_num()
```

Comment:
Is it a problem that `latest_comic_num` isnt async?

Reply:
Idk, I don't know that much about async but it hasn't given me any problems yet.